CMSgov/mint-app
# Noref/participants and interventions field corrections

## Changes and Description

- Plan Basics test_inventions --> renamed to test_interventions
- Rename providersAndParticipants on the model plan GQL object to participantsAndProviders
- All GQL enum array fields have a type definition that doesn't allow null, (so we have [] instead of null)
    - Affected task list sections
         - participants and providers
         - beneficiaries
         - ops eval and learning


## How to test this change

1. Update or create a plan basics, and set the test interventions field, confirm it works locally
2.  Make a model plan, return the task list sections, confirm the array types are empty arrays and not null
```gql
mutation createModelPlan{
  
  createModelPlan(modelName:"Excellent Model" ){
    id
    modelName
    modelCategory
    cmsCenters
    cmmiGroups
    createdBy
    createdDts
    modifiedBy
    modifiedDts
    collaborators {
      id
      euaUserID
      teamRole
      fullName
    }
    participantsAndProviders {
        id
        participants
    }
    opsEvalAndLearning {
        id
        stakeholders
        agencyOrStateHelp
    }
    beneficiaries {
        beneficiaries
    }
    # itTools {
    #     id
    #     status
    # }
    
    
  }
  
  
}
```
## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
